### DISCUSSION TITLE
The "stig=1" value on the grub boot menu not working

### DISCUSSION THREAD

**Discussion by TotieBash** [Security-Onion-Solutions/securityonion#13415] [2.4]
### Version

2.4.90

### Installation Method

Security Onion ISO image

### Description

installation

### Installation Type

Standalone

### Location

airgap

### Hardware Specs

Exceeds minimum requirements

### CPU

8

### RAM

32

### Storage for /

100

### Storage for /nsm

300

### Network Traffic Collection

tap

### Network Traffic Speeds

Less than 1Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

Yes, there are salt failures (please provide detail below)

### Logs

No, there are no additional clues

### Detail

The "stig=1" value appended on the grub menu during install does not seem to be working.  It is suppose to create separate partition as describe on the instruction but it doesn't seem to be doing it.  Please help.
https://docs.securityonion.net/en/2.4/stig.html#stig-during-the-iso-install

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment]
Stig support is a pro feature, if you're a professional services customer of ours please reach out to your rep

Does this machine have two disks? It will need two disks for ```stig=1``` to function correctly. Which will only create the additional partitions, it will not run any stig remediations.

**Reply by TotieBash** [reply]
I'm engage on email for future purchase of pro support...  Noted on 2 virutal disks, I will try this next week...

**Reply by TotieBash** [reply]
Thanks @reyesj2 the second disk is the fix for the partitioning mistery that I was after.
